source/NVDAObjects/window/winConsole.py

```diff
@@ -1,8 +1,8 @@
-#NVDAObjects/WinConsole.py
-#A part of NonVisual Desktop Access (NVDA)
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
-#Copyright (C) 2007-2019 NV Access Limited, Bill Dengler
+# NVDAObjects/WinConsole.py

```

Comment:
Please remove this line.

Reply:
Why? `NVDAObjects/behaviors.py` has a similar one as well...

Reply:
Please take a look at the [copyright headers page](https://github.com/nvaccess/nvda/wiki/Copyright-headers) especially this part is relevant:

> Note: We decided to remove the filename comment from the top of the file, since it doesn't really add anything, and is a source of error on file rename / copying copyright headers between files.